examples/local/205_clean_commerce.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright 2020 The Vitess Authors.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# this script removes the customer and corder tables from the commerce
+# keyspace
+
+vtctlclient -server localhost:15999 ApplySchema -sql-file drop_commerce_tables.sql commerce
+vtctlclient -server localhost:15999 SetShardTabletControl -blacklisted_tables=customer,corder -remove commerce/0 rdonly
```

Comment:
@sougou why do we still need to do this?

Reply:
This part of the functionality is still needed. This is what has to be moved into the vttablet itself.

Reply:
A new command will be added soon to drop tables/remove blacklists